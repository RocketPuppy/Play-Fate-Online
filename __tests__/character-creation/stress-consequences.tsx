import * as React from "react";
import * as renderer from "react-test-renderer";
import StressConsequences from "../../src/character-creation/stress-consequences";

test("renders all the pieces", () => {
  const render = renderer.create(<StressConsequences />);

  expect(render.toJSON()).toMatchSnapshot();
});
