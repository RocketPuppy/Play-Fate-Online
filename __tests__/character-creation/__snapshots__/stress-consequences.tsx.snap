// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Stress & Consequences
  </h2>
  <ul>
    <li>
      Average or Fair Physique gives you a 3-point physical stress box.
    </li>
    <li>
      Good or Great Physique gives you a 3-point and 4-point physical stress box.
    </li>
    <li>
      Superb+ Physique gives you a 3-point and 4-point physical stress box and an additional mild consequence slot.
    </li>
    <li>
      Average or Fair Will gives you a 3-point mental stress box.
    </li>
    <li>
      Good or Great Will gives you a 3-point and 4-point mental stress box.
    </li>
    <li>
      Superb+ Will gives you a 3-point and 4-point mental stress box and an additional mild consequence slot.
    </li>
  </ul>
</div>
`;
